Menu/Menu.js

```diff
@@ -1,11 +1,22 @@
 //Using jQuery, select the menu element and the menu-button element
-const menu;
-const menuButton;
+let $menu = $('.menu');
+let $menuButton = $('.menu-button');
 
 //Create a callback that will be passed to the menuButton click event listener
 //Inside this callback a class should be either added or removed from menu
-const toggleMenu = () => {
-
+let toggleMenu = () => {
+    $('.menu').toggleClass('menu--open');
 }
 
 //Using jQuery, add a click event to the menuButton, passing it the toggleMenu function.
+$('.menu-button').click(toggleMenu);
+
+$( ".menu-button" ).click(function() {
+    $( ".menu" ).animate({
+      opacity: 0.40,
```

Comment:
I noticed that the opacity is on a delay because you set the `2500` value -- just want to make sure that was intentional behavior :smile: